CHANGELOG.md

```diff
@@ -8,13 +8,69 @@ Please keep this up to date, following the [instructions](#instructions) below.
 
 ## [Unreleased](https://github.com/opendp/opendp/compare/stable...HEAD)
 
+## [0.5.0] - 2022-8-17
+[0.5.0]: https://github.com/opendp/opendp/compare/v0.4.0...v0.5.0
+
```

Comment:
You don't need to add a new heading, `release_tool.py` will promote the Unreleased section and generate a heading with the right link and date automatically.